airbyte-integrations/connectors/source-slack/acceptance-test-config.yml

```diff
@@ -16,6 +16,7 @@ tests:
   incremental:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog.json"
+      timeout_seconds: 3600
```

Comment:
:godmode:

Reply:
Looking the logs the timeout could be something around 1500~1750 :p

Reply:
marcos, i'd just keep it at 3600. It's not worth getting this number exactly right if we know this connector takes a long time